<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="AufgabenName">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="SetFinishDate">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>194</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddToCalnd">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Neue Aufgabe einfügen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Aufgabe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fertigstellungstermin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Zeit- / Datumsformatierung</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DatumsFormat">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fertigstellungstermin der gewählten Aufgabe</string>
    </property>
   </widget>
   <widget class="QLabel" name="Show_FinishDate">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Datum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choosFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Logfile wählen</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_FilePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>721</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current file Path</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="ShowWork">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
